src/processors/operation-processor.ts

```diff
@@ -22,19 +92,23 @@ export default class OperationProcessor<ResourceT = Resource> {
     });
   }
 
-  async get(op: Operation): Promise<ResourceT[]> {
+  async get(op: Operation): Promise<HasId[]> {
     return [];
   }
 
   async remove(op: Operation): Promise<void> {
     return Promise.reject();
   }
 
-  async update(op: Operation): Promise<ResourceT> {
+  async update(op: Operation): Promise<HasId> {
     return Promise.reject();
   }
 
-  async add(op: Operation): Promise<ResourceT> {
+  async add(op: Operation): Promise<HasId> {
     return Promise.reject();
   }
+
+  attributes = {}
+
+  relationships = {}
```

Comment:
Do we want to have relationships hash specific for operation in a singleton processor?

Reply:
I'm working on the `get` use case first. I think it can go forward from there.